protocols/gossipsub/src/behaviour.rs

```diff
@@ -123,6 +123,25 @@ impl Gossipsub {
         }
     }
 
+    /// lists the hashes of the topics we are currently subscribed to
```

Comment:
It would be nice if doc comments adhere to the grammar of English sentences, i.e. they begin with a capital letter and end with a full stop.